lib/services/appmanager-service.ts

```diff
@@ -48,10 +55,49 @@ class AppManagerService implements IAppManagerService {
 
 			this.$logger.info("Successfully uploaded package.");
 
-			var tamUrl = util.format("%s://%s/appbuilder/Services/tam", this.$config.AB_SERVER_PROTO, this.$config.AB_SERVER);
-			this.$logger.info("Go to %s to manage your apps.", tamUrl);
-			this.$opener.open(tamUrl);
+			this.openAppManagerStore();
+		}).future<void>()();
+	}
+
+	public openAppManagerStore(): void {
+		var tamUrl = util.format("%s://%s/appbuilder/Services/tam", this.$config.AB_SERVER_PROTO, this.$config.AB_SERVER);
+		this.$logger.info("Go to %s to manage your apps.", tamUrl);
+		this.$opener.open(tamUrl);
+	}
+
+	public publishLivePatchForAllPlatforms(): IFuture<void> {
+		return this.publishLivePatch([this.$devicePlatformsConstants.Android, this.$devicePlatformsConstants.iOS, this.$devicePlatformsConstants.WP8]);
+	}
+
+	public publishLivePatch(platforms: string[]): IFuture<void> {
+		return (() => {
+			platforms = _.map(platforms, platform => this.$mobileHelper.normalizePlatformName(platform));
+			var cachedOptionsRelease = commonOptions.release;
+			commonOptions.release = true;
+			this.configureLivePatchPlugin().wait();
+			var message = "If you have not published an AppManager LiveSync-enabled major version of this app before, you will not be able to distribute an AppManager LiveSync update for it." + os.EOL +
+				"To create a new major version enabled for AppManager LiveSync, run `$ appbuilder appmanager upload <Platform>`";
+			this.$logger.warn(message);
```

Comment:
This would print the whole message in hard-to read golden letters. Consider either printing the word "Warning" in gold, or only the first line.